internal/nlwrap/nlwrap_linux.go

```diff
@@ -5,24 +5,26 @@
 // wrapped. Functions that use the package handle need to be called as "nlwrap.X"
 // instead of "netlink.X".
 //
-// The wrapped functions currently return EINTR when NLM_F_DUMP_INTR flagged
-// in a netlink response, meaning something changed during the dump so results
-// may be incomplete or inconsistent.
+// When netlink.ErrDumpInterrupted is returned, the wrapped functions retry up to
```

Comment:
Oh! Definitely not an issue, but if you  put these things in square brackets, then `pkg.go.dev` will use them as docs-links, i.e.

```suggestion
// When [netlink.ErrDumpInterrupted] is returned, the wrapped functions retry up to
```

Will make `netlink.ErrDumpInterrupted` a clickable link to  the pkg.go.dev documentation of netlink (which is pretty neat).

(Custom reference links are also supported)